redislock.go

```diff
@@ -85,8 +84,15 @@ func (c *Client) Obtain(key string, ttl time.Duration, opt *Options) (*Lock, err
 	return nil, ErrNotObtained
 }
 
+// alia method
+func (c *Client) Lock(key string, ttl time.Duration, opt *Options) (*Lock, error) {
```

Comment:
I don't like aliases, there's not reason for it

Reply:
> I don't like aliases, there's not reason for it

The README.md example is better fix too, It has Lock().

Thank you.

Reply:
Thanks, noted.